piet-wgsl/shader/binning.wgsl

```diff
@@ -62,6 +62,7 @@ fn main(
     for (var i = 0u; i < N_SLICE; i += 1u) {
         atomicStore(&sh_bitmaps[i][local_id.x], 0u);
     }
+    workgroupBarrier();
```

Comment:
This is for the read on (new) line 110, right?

Just want to check my understanding of barriers to be honest.

Reply:
It protects the `atomicOr` on what's now line 110. This bug existed in the GLSL version too, but for some reason we never triggered it. And it *really* makes me want a race-catching tool, as that is conceptually not difficult at all, and I've been bitten so many times now.